app/assets/styles/_landing-page-styles.scss

```diff
@@ -293,10 +291,9 @@
 }
 
 .community__text{
-	.inner{
-		p{
-		 @include center($max_width: 720px);
-		}
+	margin-bottom:6em;
+	p{
+	 @include center($max_width: 720px);
 	}
 }
```

Comment:
Please mind the text indentation.
- Should be set to 2 spaces
- There should be a space between the text and `{` and between `:` and property value.

It helps with readability.